webapp/platform/types/src/config.ts

```diff
@@ -62,6 +62,7 @@ export type ClientConfig = {
     EnableUserStatuses: string;
     EnableLastActiveTime: string;
     EnableTimedDND: string;
+    EnableCrossTeamSearch: 'true' | 'false';
```

Comment:
ì see this is a pattern in some parts of this file to not convert to a boolean, is there a reason for this?